README.md

```diff
@@ -61,6 +61,17 @@ php bin/magento module:enable ADM_QuickDevBar
 php bin/magento setup:upgrade
 ```
 
+## With Modman
+
+In the Magento root directory
+
+- Install the module
+```
+modman clone git@github.com:mgtcommerce/Mgt_Developertoolbar.git
```

Comment:
There is a little confusion on the repository name ;-)
```suggestion
modman clone git@github.com:vpietri/magento2-developer-quickdevbar.git
```

Reply:
@vpietri Fixed in a new commit.